### ISSUE TITLE
Show warning squiggle if users edit a file inside venv

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/pylance-release#6706]
It seems like some users place their project code inside the virtual environments. This is bad design based on virtual environment guidelines. New users of Python seem to follow this since it makes it easier to import. Guide them to better practices.

**Comment by rchiodo** [comment]
Wait, they create a virtual env and then start adding their own packages to the site-packages folder?

**Comment by karthiknadig** [comment]
Yes. That is correct. I keep having to point out that this is bad design.

**Comment by debonte** [comment]
Moving this issue to discussions as an enhancement request for comments and upvotes.
